Libbna/CUSTOM-CMS
# #231: Article body ckeditor issue.

* Solved the ckeditor issue.

* Steps for testing : 
   * Add a new article and use the ckeditor options.
   * The ckeditor changes should reflect in article-content page. 
   * Follow the same steps for edit article.
